verl/trainer/ppo/ray_trainer.py

```diff
@@ -954,6 +954,8 @@ def fit(self):
                             self.async_rollout_manager.sleep()
                         timing_raw.update(gen_batch_output.meta_info["timing"])
                         gen_batch_output.meta_info.pop("timing", None)
+                    # TODO (Issue 5): Remove continue
```

Comment:
nit: Add a link to the issue and explain that the you're planning to remove the continue when implementing Actor and Critic.